scripts/funcTests/runFuncTests.sh

```diff
@@ -157,4 +157,7 @@ esac
 
 popd
 
+# Cleanup TMP directory
+rm "/tmp/"dotnet.* -rf
```

Comment:
Perhaps move this to just after dotnet-install.sh is run, so the exit on line 122 and 79 won't prevent this cleanup.

Reply:
Makes sense assuming that nothing else in the script relies on anything in TMP. Went ahead and moved it up.

Reply:
fwiw NuGet itself relies on the temp directory, but as long as you don't delete it in the middle of the tests running, it's not a big deal.

Reply:
plus this line isn't deleting everything from /tmp, just files we know were created by `dotnet-install.sh`. I've never seen any other file use the same name pattern.

Reply:
oh I missed that. 
Why not delete everything? 
That's a common issue with the tmp folder in linux where a restart is required, compared to windows 10 where there's already a service that cleans it up.